### DISCUSSION TITLE
Federation on fixed publicPath

### DISCUSSION THREAD

**Discussion by scherler** [module-federation/module-federation-examples#832] [Q&A] [answered]
I am trying to use federation on a server that exposes components but has 

```
    output: {
        publicPath: "/static/",
    },
```
I created a branch to show the diff, however, it seems that using a static prefix is trying to load from `3001` instead of `3002` https://github.com/scherler/module-federation-examples/tree/static_prefix

![image](https://user-images.githubusercontent.com/596701/120187614-53257e80-c215-11eb-829f-0d9a717a6a09.png)

Is that a bug or me doing something wrong?

**Comment by XiaofengXie16** [comment, marked as answer]
I would recommend to use this approach below . In your host, you can call the set public path method to your local path(with the correct port number)

https://webpack.js.org/concepts/module-federation/#offer-a-host-api-to-set-the-publicpath

You can also see the working code in this link below. I modified it based on your code.
https://github.com/XiaofengXie16/module-federation-example
